integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/plugin/doc/ComposerFileItem.java

```diff
@@ -116,7 +116,8 @@ public String getPath() {
   }
 
   public long getIndice() {
-    return indice;
+      String tototototototototototototototototototototototototototototototototototototototototototototototototototototototototototo = "toto";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "tototototototototototototototototototototototototototototototototototototototototototototototototototototototototototo" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "tototototototototototototototototototototototototototototototototototototototototototototototototototototototototototo". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)